src/js/angular/clustermanagement/controllers/cluster-management.controller.js

```diff
@@ -86,7 +86,9 @@ function ClusterManagementCtrl($scope, $http, $q, toastr, $repositories, $uibMod
     };
 
     $scope.showUpdateClusterGroupDialog = () => {
+        $scope.setLoader(true);
         getLocationsWithRpcAddresses().then(() => {
+            $scope.setLoader(false);
```

Comment:
If the request fails who will remove the loader?

Reply:
added catch to handle scenario